### ISSUE TITLE
Missing Tests

### ISSUE THREAD

**Issue by adamvandenhovenfm** [chovy/humanparser#14]
This is a pretty good parser but it needs some more exhaustive tests:

``` javascript
{
    name: 'Mr. Hearst',
    result: {
        salutation: 'Mr.',
        firstName: '',
        middleName: '',
        lastName: 'Hearst',
        suffix: '',
        fullName: 'Mr. Hearst'
    }
},
```

**Comment by adamvandenhovenfm** [comment]
oops. I just realized I wasn't paying attention. This is the one that's missing.

**Comment by chovy** [comment]
can you submit a PR and fix if necessary?

**Comment by chovy** [comment]
fixed this in v1.3.0
